packages/build/CHANGELOG.md

```diff
@@ -5,7 +5,8 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
-<!-- ## Unreleased -->
+## Unreleased
+* Fix AMD transform bug where if an HTML document had multiple type=module scripts, and any of them (apart from the first) had any kind of import, then that import was not accessible (because it was mapped to the wrong module callback function argument).
```

Comment:
What about a policy where we update all affected packages' changelogs when there's important bugfixes or new features? So in this case we'd copy this change notice to CLI and Polyserve too.

That'll also help `lerna publish` infer when it should stamp a new release of a package.

Reply:
Done.